flexmeasures/conftest.py

```diff
@@ -109,12 +109,18 @@ def create_test_db(app):
 
 @pytest.fixture(scope="module")
 def setup_accounts(db) -> dict[str, Account]:
-    return create_test_accounts(db)
+    test_accounts = create_test_accounts(db)
+    test_accounts |= add_consult_account_access_to_prosumer_account(db, test_accounts)
+    return test_accounts
 
 
 @pytest.fixture(scope="function")
 def setup_accounts_fresh_db(fresh_db) -> dict[str, Account]:
-    return create_test_accounts(fresh_db)
+    test_accounts = create_test_accounts(fresh_db)
+    test_accounts |= add_consult_account_access_to_prosumer_account(
```

Comment:
This operator is not supported in Python 3.8, see https://github.com/FlexMeasures/flexmeasures/actions/runs/6549108023/job/17785337960?pr=877

Anyway, I think we don't need to do it like this, see my other comment. Then this problem would also go away.

Reply:
fixed `create_test_accounts()` to include the consultanct account and therefore was able to change this function back to the original version. This also fixes Python 3.8 support